Assets/Scripts/Scenes/InGame/Player/TempPlayerMove.cs

```diff
@@ -148,7 +148,7 @@ void Update()
 
             //���R����
             if (this.gameObject.transform.position.y > 0)
-                this.gameObject.transform.position -= new Vector3(0, 9.8f *Time.deltaTime, 0);
+                _characterController.Move(new Vector3(0, -9.8f * Time.deltaTime, 0) * moveSpeed * 10 * Time.deltaTime);

```

Comment:
```suggestion
                _characterController.Move(new Vector3(0, -9.8f * Time.deltaTime, 0));
```
試してみましたがこっちの方が自然な自由落下だと思います。